dd-trace-ot/src/main/java/datadog/opentracing/decorators/DBStatementAsResourceName.java

```diff
@@ -23,15 +24,9 @@ public boolean shouldSetTag(final DDSpanContext context, final String tag, final
     }
 
     // Assign service name
-    if (!super.shouldSetTag(context, tag, value)) {
-      // TODO: remove properly the tag (immutable at this time)
-      // the `db.statement` tag must be removed because it will be set
-      // by the Datadog Trace Agent as `sql.query`; here we're removing
-      // a duplicate that will not be obfuscated with the current Datadog
-      // Trace Agent version.
-      context.setTag(Tags.DB_STATEMENT.getKey(), null);
-      return false;
-    }
+    // if (!super.shouldSetTag(context, tag, value)) {
+    //   return true;
```

Comment:
Instead of commenting out sections of code, would it make more sense to write the intended behavior and then provide a rationale?